infra/Pos/Diffusion/Types.hs

```diff
@@ -8,26 +8,26 @@ module Pos.Diffusion.Types
     , dummyDiffusionLayer
     ) where
 
-import           Universum
-import           Data.Map.Strict                  (Map)
-import qualified Data.Map.Strict                  as Map
-import           Formatting                       (Format)
+import           Data.Map.Strict (Map)
+import qualified Data.Map.Strict as Map
+import           Formatting (Format)
+import           GHC.Generics (Generic)
 import           Pos.Communication.Types.Protocol (NodeId)
-import           Pos.Core.Block                   (Block, BlockHeader, MainBlockHeader)
-import           Pos.Core                         (HeaderHash, ProxySKHeavy)
-import           Pos.Core.Txp                     (TxAux)
-import           Pos.Core.Update                  (UpId, UpdateVote, UpdateProposal)
-import           Pos.Reporting.Health.Types       (HealthStatus (..))
-import           Pos.Core.Ssc                     (Opening, InnerSharesMap, SignedCommitment,
-                                                   VssCertificate)
-import           Pos.Util.Chrono                  (OldestFirst (..))
+import           Pos.Core (HeaderHash, ProxySKHeavy)
+import           Pos.Core.Block (Block, BlockHeader, MainBlockHeader)
+import           Pos.Core.Ssc (InnerSharesMap, Opening, SignedCommitment, VssCertificate)
+import           Pos.Core.Txp (TxAux)
+import           Pos.Core.Update (UpId, UpdateProposal, UpdateVote)
+import           Pos.Reporting.Health.Types (HealthStatus (..))
+import           Pos.Util.Chrono (OldestFirst (..))
+import           Universum
```

Comment:
If we're reformatting the imports, let's follow [the style guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md#imports) on it.

Reply:
This is done automatically by my editor following what's in the `.stylish-haskell.yaml` file. I see that the newlines are missing to separate the different groups (I think that's what you're referring to :o) 

Even though I like the style, I have to say that if we can't enforce style with a linter, it's probably not worth enforcing at all :/ ... 
I'll isolate Universum back to the top and leave a blank line after it. I truly believe however that we have to find a way to automatically enforce that.

Reply:
There are lots of things we can't enforce with a linter (yet) due to tooling issues. We are humans, after all, we get the final say in this programming thing :)